.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,14 +8,26 @@ Rules:
    a) MUST match one of the following patterns:
       ^Issue #\\d+: .*$
       ^Pull #\\d+: .*$
-      ^(minor|config|infra|doc|spelling|dependency): .*$
+      ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    b) MUST contain only one line of text
    c) MUST NOT end with a period, space, or tab
    d) MUST be less than or equal to 200 characters
 
 To avoid multiple iterations of fixes and CIs failures, please read
 https://checkstyle.org/contributing.html
 
+_______________________________________________________
+
+Check regression report generation template (see details [here](https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#report-generation)):
```

Comment:
I am not sure it is good.
Big template is cad, as people will not read it.

Reply:
All good points, and I agree. This could confuse new contributors making PR's that do not need report generation, also. This was a PR to enable my own laziness, I think :)